doc/generate-kuadrant-auth-policy.md

```diff
@@ -315,9 +315,39 @@ EOF
 
 </details>
 
-> Replace `${KEYCLOAK_PUBLIC_DOMAIN}` with your SSO instance domain
+> Replace `${KEYCLOAK_ISSUER}` with your SSO instance issuer endpoint for your `petstore` realm. 
```

Comment:
What do you think about moving this to be above the resource that uses it so they know to set it before they create the resource it falls more in line with other docs we have